src/chapters/control-flow-and-collections/list.rst

```diff
@@ -0,0 +1,343 @@
+
+To demonstrate the use of these methods, we'll create a new ``List``
+called ``planets``.
+
+.. sourcecode:: csharp
+
+   List<string> planets = new List<string>();
+
+Ok, we've got an empty List. We need to use the class's ``.Add()`` method
+to populate this collection with items.
+
+.. admonition:: Note
+
+   There are other means to declare and initialize an List in fewer lines.
+   These require knowledge of other collections types, so we'll stick with ``.Add()``
+   for the time being.
+
+Using ``.Add()`` to populate ``planets``:
+
+.. sourcecode:: C#
+   :linenos:
+
+   planets.Add("Mercury");
+   planets.Add("Venus");
+   planets.Add("Earth");
+   planets.Add("Mars");
+   planets.Add("Jupiter");
+   planets.Add("Saturn");
+   planets.Add("Uranus");
+   planets.Add("Neptune");
+
+Thus, the first item in this table:
+
+.. _list-methods:
+
+.. list-table:: List methods in C#
+   :header-rows: 1
+
+   * - C# Syntax
+     - Description
+     - Example
+   * - ``Add()``
+     - Adds an item to the List
+     - ``planets.Add("Pluto")`` adds ``Pluto`` to ``planets``
+   * - ``Count``
+     - Returns the number of items in an List, as an ``int``
+     - ``planets.Count`` returns ``9``
+   * - ``Contains()``
+     - Checks to see if the List contains a given item, returning a Boolean
+     - ``planets.Contains("Earth")`` returns ``true``
+   * - ``IndexOf()``
+     - Looks for an item in an List, returns the index of the first occurrence of the item if it exists, returns -1 otherwise
+     - ``planets.IndexOf("Jupiter")`` returns ``4``
+
+Here's a couple more methods that require slightly longer descriptions:
+
+.. _listsort:
+
+.. list-table:: List.Sort()
+   :header-rows: 1
+
+   * - C# Syntax
+     - Description
+     - Example
+   * - ``.Sort()``
+     - Rearranges the elements of an ``List`` into ascending order.
+     - ``planets.Sort()`` produces ``["Earth", "Jupiter", "Mars", "Mercury", "Neptune", "Pluto", "Saturn", "Uranus", "Venus"]``
+
+.. list-table:: ToArray()
+   :header-rows: 1
+
+   * - C# Syntax
+     - Description
+     - Example
+   * - ``ToArray()``
+     - Returns an Array containing the elements of the List
+     - ``planets.ToArray()`` returns
+       ``{"Earth", "Jupiter", "Mars", "Mercury", "Neptune", "Pluto", "Saturn", "Uranus", "Venus"}``
+
+Perhaps you recall that in C#, you must know the size of the Array when you
+create it. So we'll need to first define the new Array before we can use
+``toArray()``.
+
+.. sourcecode:: csharp
+   :linenos:
+
+   string[] planetsArray = new string[planets.Count];
+   planetsArray = planets.ToArray();
+
+Speaking of Arrays, let's see the Array version of Gradebook next.
+
+Check Your Understanding
+-------------------------
+
+.. admonition:: Question
+
+   The number of entries in an ``List`` may not be modified.
+
+   #. True
+   #. False
+
+.. ans: False
+
+.. admonition:: Question
+
+   Create an ``List`` called ``charStars`` containing ``a``, ``b``, and ``c``.
```

Comment:
an list...

Reply:
also flagging this question in case you decide to add an explanation of how to declare and initialize a list in the same line, you may want to add another correct answer here?

Reply:
oh and .add needs to be .Add

Reply:
Added that syntax, but kept the answers the same.